content/webapp/services/prismic/transformers/card.ts

```diff
@@ -3,14 +3,13 @@ import { transformImage } from '@weco/common/services/prismic/transformers/image
 import { Card } from '../../../types/card';
 import { asText, asTitle, transformFormat } from '.';
 import { transformLink } from '@weco/common/services/prismic/transformers';
-import { isFilledLinkToDocument } from '@weco/common/services/prismic/types';
 
 export function transformCard(document: CardPrismicDocument): Card {
   const { title, description, image, link } = document.data;
 
   return {
     type: 'card',
-    id: isFilledLinkToDocument(link) ? link.id : undefined,
```

Comment:
I can't be sure why this was as it was – I _think_ it was probably a mistake. And it was the reason why `id` would be undefined [here](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/components/Body/Body.tsx#L468) and AsyncSearchResults could end up querying Prismic without any parameters (which then sends back a lot, and potentially breaks the frontend which wasn't expecting it)